src/Ocelot.Cache.CacheManager/Ocelot.Cache.CacheManager.csproj

```diff
@@ -28,11 +26,14 @@
     <ProjectReference Include="..\Ocelot\Ocelot.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="StyleCop.Analyzers" Version="1.1.118">
+    <PackageReference Include="StyleCop.Analyzers" Version="1.2.0-beta.66">
```

Comment:
No final version for this one compatible with .net core 3?

Reply:
yes